### PR TITLE
Rs 2022 04 09

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 1]
```diff
@@ -32,42 +33,45 @@ or maybe we're just here because that devilish smile pushed me here.
...
 -- 部活動禁止と早期下校の通達のおかげで、正門に<人気|ひとけ>は皆無だった。
-Thanks to the ban on club activities and the notification to go home early, there is virtually no sign of life at the entrance.
+There's virtually no signs of life at the main gate,
+ no doubt thanks to the official notice stating that all club activities are suspended and students are to return home promptly.
 }
 [sha:f3abdacdc441e451d3e2e6be50a441389ea51461]{
 -- Page 398, Offset 25270.
 -- この分なら人が残っていたとしても片手で数えるほどだろう。目立つ事さえしなければ、中庭ぐらいまでは案内できそうだ。
-At this rate, even if there are people remaining, I would probably be able to count them on one hand. As long as we don't do anything to stand out, I should be able to show her to the courtyard at least.
+If there are any people lingering around, I would probably be able to count them on one hand.
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:42]: if there were

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 2]
```diff
@@ -142,27 +152,30 @@ It's just as Arcueid said: there really isn't anyone here but us.
 [sha:a8021f316174b04a8db86aa56455fc6639e9fbeb]{
 -- Page 406, Offset 25291.
 -- アルクェイドはじっと上目遣いで見つめてくる。……なんとなく、アルクェイドの言いたい事が分かってしまった。
-Arcueid stares at me with upturned eyes... Somehow, I already knew what Arcueid wanted to say.
+// Literal 上目遣い doesn't translate super well imo,
+// compare image search results of the JP and literal EN
+Arcueid stares at me with puppy dog eyes...
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:157]: "Arcueid stares at me with upturned eyes..." reads perfectly naturally to me imo. With that said though, I think this version is also fine

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 3]
```diff
@@ -329,7 +348,8 @@ I open the glass window and enter the school building without taking off my dirt
 [sha:4ae34615b2cba27f790ff7770681aa087c412f2a]{
 -- Page 420, Offset 25328.
 -- まあ、予測通りというか、お約束というか。
-Well, it's just as predicted. Or perhaps just as promised.
+// _お_約束 can mean like 'expected/predictable gag, cliche' which I assume is what's intended here
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:351]: although i don't take issue with this TL, I'm not actually sure what he's referring to with お約束. Is it him not taking off his shoes? is it Arcueid wanting to go to his classroom?

File: script/Arcueid/Day 10/10_00_ARC10_5.txt [hunk 4]
```diff
@@ -450,7 +471,9 @@ Out of all the questions she's asked so far, this one's the most difficult to gr
 [sha:92f89ce99beb0b27f9f3d4761cb42a6cf24578bf]{
 -- Page 428, Offset 25352.
 -- 「ここで習った知識や技術は、あなたの人生において一度も使われない事もあるのよね。そういうのって無駄じゃないの？」
-"There are knowledge and skills you learn here that you'll never use once in your entire life, right? Don't you think that's pointless?"
+// Nasu still salty about having to learn calculus in school
```
Comment (Avuxo) [inline script/Arcueid/Day 10/10_00_ARC10_5.txt:474]: he's clearly never learned anything he learned in literature classes either

**Review by Avuxo — APPROVED** [review]
I totally forgot to TLC the QA file the first time around, sorry lol. Nice work on these

File: script/Arcueid/Day 10/10_00_ARC10_5.txt (hunk 3 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 10/10_00_ARC10_5.txt:351]: Also the next line is 
> The very first place that Arcueid wanted to go was my classroom on the third floor.

but it was hidden below the fold on the diff

**Pull request merged**